### ISSUE TITLE
Help creating toolbar

### ISSUE THREAD

**Issue by AlexandreFQM** [xmartlabs/Eureka#728]
Hi guys,

Could you help me how can I add a toolbar using eureka?

Thanks.

**Comment by mtnbarreto** [comment]
Is your form (`FormViewCotroller`) embedded into a UINavigationController?

**Comment by mtnbarreto** [comment]
Eureka does not manage any toolbar. You can do it by following Apple [reference](https://developer.apple.com/library/content/documentation/UserExperience/Conceptual/UIKitUICatalog/UIToolbar.html). 

You can also add a toolbar to your FormViewController through interface builder and show the table view below. 

> Take into account that you will have to connect the  FormViewController table view outlet using a storyboard if you are not already doing this. You will also have to set up toolbar, table view constraints.

**Comment by AlexandreFQM** [comment]
I tried to add the toolbar in the form view controller using the interface however it does not appear.

It is structured in this way:

Navigation View Controller > FormViewController (UIView) and inside the UIView in the storyboard was placed a toolbar and some buttom bar item.

I think I'm doing something wrong, would you have an example to show?

<img width="1009" alt="formviewcontroller" src="https://cloud.githubusercontent.com/assets/10813893/20610398/37a8d7f0-b280-11e6-8deb-1ceb0745e09f.png">

NavigationViewController Attributes

<img width="262" alt="navigationcontroller" src="https://cloud.githubusercontent.com/assets/10813893/20610399/37ac26e4-b280-11e6-813d-51f74b4e8d18.png">


Thank you

**Comment by mats-claassen** [comment]
So you have a UIViewController in the Storyboard but you have not added a `tableView` to it right? It is recommended that you do and then connect it to the `tableView` variable of the FormViewController.

Did you inspect the view hierarchies? Maybe the toolbar is just hidden behind the `tableView`

**Comment by AlexandreFQM** [comment]
@mats-claassen 

Sorry for the delay in returning.

Now, I have a view controller> TableView and connected with the FormViewController variable. It worked out here.

But I'm having problems with the constraints, the toolbar is simply not in the footer, as the images below.

Here we see the toolbar
<img width="379" alt="captura de tela 2016-12-06 as 16 43 14" src="https://cloud.githubusercontent.com/assets/10813893/20938702/41584414-bbd3-11e6-892a-0721d772b617.png">

Here I changed tab
<img width="377" alt="captura de tela 2016-12-06 as 16 43 27" src="https://cloud.githubusercontent.com/assets/10813893/20938703/4159189e-bbd3-11e6-940b-cfd59c377957.png">

Here I went back to the page with the toolbar and it automatically adjusts
<img width="378" alt="captura de tela 2016-12-06 as 16 43 33" src="https://cloud.githubusercontent.com/assets/10813893/20938704/415b265c-bbd3-11e6-9b93-b72ff5472011.png">
